generator/cmd/protoc-gen-gclient/main_test.go

```diff
@@ -0,0 +1,59 @@
+package main
```

Comment:
nit: here and in all source code files, please include the copyright boilerplate.

Reply:
done.